### ISSUE TITLE
IPMI shanaggins

### ISSUE THREAD

**Issue by llua** [cseader/home-conf#1]
Not sure how non-important this repo is, but it contains valid passwords for publicly accessible BMCs in the .bash_history which might not be desired. along with ssh private key to some hosts.

**Comment by neutronscott** [comment]
oops

**Comment by cseader** [comment]
Yeah. Someone has unfortunately forked it and now I can't remove that fork. ideas? I have deleted those files from my project. they never should have been there in the first place

**Comment by neutronscott** [comment]
Change your password! It's compromised.

https://help.github.com/articles/remove-sensitive-data/

**Comment by cseader** [comment]
Passwords are no longer valid.
